deps/rabbit/src/rabbit_fifo.erl

```diff
@@ -456,11 +460,13 @@ apply(#{system_time := Ts} = Meta, {down, Pid, noconnection},
     % Monitor the node so that we can "unsuspect" these processes when the node
     % comes back, then re-issue all monitors and discover the final fate of
     % these processes
+
+    NotifyEffect = notify_decorators_effect(State),
```

Comment:
what is the scenario we are notifying here?

Reply:
Consumers are marked as suspected down, so it could end up with no active consumers at all and that needs to be notified.